internal/provider/resource_tfe_policy.go

```diff
@@ -283,7 +283,7 @@ func resourceTFEPolicyUpdate(d *schema.ResourceData, meta interface{}) error {
 		path := d.Get("name").(string) + ".sentinel"
 		vKind, ok := d.GetOk("kind")
```

Comment:
Alternatively:
```go
vKind, ok := d.GetOK("kind").(string)
...
```
It also seems to be cast again in L313 under `if d.HasChange("policy") {}`. Maybe we can consolidate this a bit.

Reply:
Done